packages/sanity/src/structure/panes/document/inspectDialog/InspectDialog.tsx

```diff
@@ -27,7 +27,6 @@ export function InspectDialog(props: InspectDialogProps) {
   where the inspect dialog lives.
   This also means that when a page is loaded, the state of the tabs remains and doesn't revert to the pane tab */
   const [viewModeId, onViewModeChange] = useStructureToolSetting(
-    'structure-tool',
```

Comment:
Don't have full context if it's used in multiple places, but the function signature has not changed, won't `inspect-view-preferred-view-mode-${paneKey}` become the namespace here? Is that expected?

Reply:
Good point -- so it looks like the behavior we have in the current version of the studio makes a key like `structure-tool::structure-tool::inspect-view` (`useStructureToolSetting` always added the `structure-tool` namespace). It seems like this was an error.

I can restore this to have the old duplicative behavior to not break anything and we can migrate it when we have a strategy to migrate all old keys.